packages/core/admin/admin/src/pages/Settings/pages/Roles/components/ConditionsModal.tsx

```diff
@@ -240,7 +240,7 @@ const ActionRow = ({
 
   return (
     <Flex tag="li" background={isGrey ? 'neutral100' : 'neutral0'} paddingBottom={3} paddingTop={3}>
-      <Flex paddingLeft={6} style={{ width: 180 }}>
+      <Flex paddingLeft={6} paddingRight={1} style={{ width: 180 }}>
```

Comment:
```suggestion
        <Flex
      tag="li"
      background={isGrey ? 'neutral100' : 'neutral0'}
      paddingBottom={3}
      paddingTop={3}
      justifyContent={'space-evenly'}
    >
      <Flex style={{ width: 180 }}>
```

Reply:
@jhoward1994 thanks for suggestion. Added suggested changes please check